mender-convert-modify

```diff
@@ -193,6 +193,7 @@ cat <<- EOF > work/mender.conf.data
 EOF
 
 run_and_log_cmd "sudo cp work/mender.conf.data work/rootfs/data/mender/mender.conf"
+run_and_log_cmd "chmod 600 work/rootfs/data/mender/mender.conf"
```

Comment:
I think you need `sudo` here.

Reply:
> I think you need `sudo` here.

ah, crap, copy-pasted from the line above, and removed sudo by mistake. [thanks](https://github.com/mendersoftware/mender-convert/pull/219/commits/564d263f53be0cc4b0b69e3f45f6153541462ae3)